src/library/scala/Byte.scala

```diff
@@ -59,7 +59,7 @@ final abstract class Byte private extends AnyVal {
   *         filling in the new right bits with zeroes.
   * @example {{{ 6 << 3 == 48 // in binary: 0110 << 3 == 0110000 }}}
   */
-  @deprecated("shifting a value by a `Long` argument is deprecated (except when the value is a `Long`).\nCall `toInt` on the argument to maintain the current behavior and avoid the deprecation warning.", "2.12.7")
+  @deprecated("Shifting a value by a `Long` argument is deprecated (except when the value is a `Long`).\nCall `toInt` on the argument to maintain the current behavior and avoid the deprecation warning.", "2.12.7")
```

Comment:
The files for primitive types are generated - you need to change `project/GenerateAnyVals.scala`